graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/execution/SubscriptionHandler.kt

```diff
@@ -36,21 +36,21 @@ interface SubscriptionHandler {
     /**
      * Execute GraphQL subscription request and return a Reactor Flux Publisher that emits 0 to N [GraphQLResponse]s.
      */
-    fun executeSubscription(graphQLRequest: GraphQLRequest): Flux<GraphQLResponse>
+    fun executeSubscription(graphQLRequest: GraphQLRequest): Flux<GraphQLResponse<*>>
 }
 
 /**
  * Default implementation of GraphQL subscription handler.
  */
 open class SimpleSubscriptionHandler(private val graphQL: GraphQL) : SubscriptionHandler {
 
-    override fun executeSubscription(graphQLRequest: GraphQLRequest): Flux<GraphQLResponse> = Mono.subscriberContext()
+    override fun executeSubscription(graphQLRequest: GraphQLRequest): Flux<GraphQLResponse<*>> = Mono.subscriberContext()
         .flatMapMany { reactorContext ->
             val graphQLContext = reactorContext.getOrDefault<Any>(GRAPHQL_CONTEXT_KEY, null)
             graphQL.execute(graphQLRequest.toExecutionInput(graphQLContext))
                 .getData<Publisher<ExecutionResult>>()
                 .toFlux()
                 .map { result -> result.toGraphQLResponse() }
-                .onErrorResume { error -> Flux.just(GraphQLResponse(errors = listOf(SimpleKotlinGraphQLError(error)))) }
+                .onErrorResume { error -> Flux.just(GraphQLResponse<Any>(errors = listOf(GraphQLError(error.message.orEmpty())))) }
```

Comment:
Do we want to return `SimpleKotlinGraphQLError` here as well?

Reply:
`GraphQLResponse` need the generic `GraphQLError` from the new types. So we can't use 
`SimpleKotlinGraphQLError` but we could call `SimpleKotlinGraphQLError().toGraphQLKotlinType()`

Reply:
are you sure? we used to return the simple kotlin error

Reply:
We were returning a `GraphQLResponse` and the old/current `GraphQLResponse.errors` is `graphql.GraphQLError` type. We no longer need that. We are using the generic type

https://github.com/ExpediaGroup/graphql-kotlin/blob/d60e4391cda19325c95b5dc724e090555910bf5d/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/model/GraphQLResponse.kt#L28